lib/active_merchant/billing/gateways/safe_charge.rb

```diff
@@ -73,12 +73,16 @@ def refund(money, authorization, options = {})
         add_transaction_data('Credit', post, money, options.merge!({ currency: original_currency }))
         post[:sg_CreditType] = 2
         post[:sg_AuthCode] = auth
-        post[:sg_TransactionID] = transaction_id
         post[:sg_CCToken] = token
         post[:sg_ExpMonth] = exp_month
         post[:sg_ExpYear] = exp_year
 
-        commit(post)
+        if options[:unreferenced_refund] == true
```

Comment:
Instead of duplicating the `commit(post)` statement with this logic, could you instead add an `unless` to the end of line 76?

`post[:sg_TransactionID] = transaction_id unless options[:unreferenced_refund]`

I think this would make the change a little bit more DRY

Reply:
yep I totally agree! I was completely brain fogged and was already fixing it when the linter failed lol, thanks!